### PR TITLE
Update test infra deps script

### PR DESCRIPTION
**Pull request by matzew**
Fixes #

## Proposed Changes

- updating the infra deps, steps: see on commit messages

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -18,49 +18,10 @@ set -o errexit
...
-rm -rf $(find vendor/ -name 'BUILD')
-rm -rf $(find vendor/ -name 'BUILD.bazel')
-rm -rf $(find vendor/ -name '*_test.go')
-
-export GOFLAGS=-mod=vendor
-
-update_licenses third_party/VENDOR-LICENSE "./..."
+# Patch k8s leader-election fixing graceful release
+# More information: https://github.com/kubernetes/kubernetes/pull/91942
+git apply ${REPO_ROOT_DIR}/hack/k8s-client-go.patch
```
Comment (n3wscott) [inline hack/update-deps.sh:27]: do you need this patch? you did not use it before.

Comment (slinkydeveloper) [reply hack/update-deps.sh:27]: we applied it to eventing-kafka-broker too https://github.com/knative-sandbox/eventing-kafka-broker/pull/293

Comment (matzew) [reply hack/update-deps.sh]: based on feeback, I removed that for now

Thanks @n3wscott

**Comment by n3wscott** [comment]
Ping @matzew , you need to run the update script

**Comment by matzew** [comment]
done (I thought I did...) :man_shrugging:

**Comment by matzew** [comment]
/close

go home you are drunk

**Pull request closed without merging**